code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -396,6 +396,11 @@ Contains:
 	if (user.head == src)
 		user.faction -= "carp"
 
+/obj/item/clothing/suit/space/hardsuit/carp/old
```

Comment:
this is objectively better than the curator's spacesuit kit. Perhaps give that kit a void jetpack to compensate?